### PR TITLE
[MBL-16915][Student] Quizzes screen letter grade only

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: quiz points should be hidden if letter grade only.

refs: MBL-16915
affects: Student
release note: none

## Checklist

- [x] Tested in dark mode
- [x] Tested in light mode

### CODE REVIEW

**Review by kdeakinstructure — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by kdeakinstructure** [comment]
QA 👍

File: automation/espresso/src/main/kotlin/com/instructure/canvas/espresso/mockCanvas/endpoints/CourseEndpoints.kt [hunk 1]
```diff
@@ -571,11 +571,7 @@ object CourseQuizListEndpoint : Endpoint(
         response = {
             GET { // Get the list of quizzes for course
                 val quizzesForCourse = data.courseQuizzes[pathVars.courseId]
-                if (quizzesForCourse != null) {
-                    request.successResponse(quizzesForCourse)
-                } else {
-                    request.unauthorizedResponse()
-                }
+                request.successResponse(quizzesForCourse.orEmpty())
```
Comment (tamaskozmer) [inline automation/espresso/src/main/kotlin/com/instructure/canvas/espresso/mockCanvas/endpoints/CourseEndpoints.kt:574]: What's the reason for this change?

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Pull request merged**